### ISSUE TITLE
Xlib module used in pyautogui but not listed as a dependency in setup.py

### ISSUE THREAD

**Issue by jtara1** [asweigart/pyautogui#210]
Installing xlib myself thru pip fixed this. I can create a PR to fix this for this module.

The code below shows me:
1. failing to install pyautogui b/c xlib not installed
2. installing xlib myself with pip
3. install pyautogui successfully

```
j@j-pc ~ $ sudo pip3.5 install pyautogui
The directory '/home/j/.cache/pip/http' or its parent directory is not owned by the current user and the cache has been disabled. Please check the permissions and owner of that directory. If executing pip with sudo, you may want sudo's -H flag.
The directory '/home/j/.cache/pip' or its parent directory is not owned by the current user and caching wheels has been disabled. check the permissions and owner of that directory. If executing pip with sudo, you may want sudo's -H flag.
Collecting pyautogui
  Downloading PyAutoGUI-0.9.36.tar.gz (46kB)
    100% |████████████████████████████████| 51kB 3.2MB/s 
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-830e71j3/pyautogui/setup.py", line 6, in <module>
        version=__import__('pyautogui').__version__,
      File "/tmp/pip-build-830e71j3/pyautogui/pyautogui/__init__.py", line 115, in <module>
        from . import _pyautogui_x11 as platformModule
      File "/tmp/pip-build-830e71j3/pyautogui/pyautogui/_pyautogui_x11.py", line 7, in <module>
        from Xlib.display import Display
    ImportError: No module named 'Xlib'
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-830e71j3/pyautogui/
j@j-pc ~ $ sudo pip3.5 install Xlib
The directory '/home/j/.cache/pip/http' or its parent directory is not owned by the current user and the cache has been disabled. Please check the permissions and owner of that directory. If executing pip with sudo, you may want sudo's -H flag.
The directory '/home/j/.cache/pip' or its parent directory is not owned by the current user and caching wheels has been disabled. check the permissions and owner of that directory. If executing pip with sudo, you may want sudo's -H flag.
Collecting Xlib
  Downloading xlib-0.21-py2.py3-none-any.whl (123kB)
    100% |████████████████████████████████| 133kB 3.8MB/s 
Requirement already satisfied: six>=1.10.0 in /usr/local/lib/python3.5/dist-packages (from Xlib)
Installing collected packages: Xlib
Successfully installed Xlib-0.21
j@j-pc ~ $ sudo pip3.5 install pyautogui
The directory '/home/j/.cache/pip/http' or its parent directory is not owned by the current user and the cache has been disabled. Please check the permissions and owner of that directory. If executing pip with sudo, you may want sudo's -H flag.
The directory '/home/j/.cache/pip' or its parent directory is not owned by the current user and caching wheels has been disabled. check the permissions and owner of that directory. If executing pip with sudo, you may want sudo's -H flag.
Collecting pyautogui
  Downloading PyAutoGUI-0.9.36.tar.gz (46kB)
    100% |████████████████████████████████| 51kB 2.7MB/s 
Collecting pymsgbox (from pyautogui)
  Downloading PyMsgBox-1.0.6.zip
Collecting PyTweening>=1.0.1 (from pyautogui)
  Downloading PyTweening-1.0.3.zip
Requirement already satisfied: Pillow in /usr/local/lib/python3.5/dist-packages (from pyautogui)
Collecting pyscreeze (from pyautogui)
  Downloading PyScreeze-0.1.13.tar.gz
Installing collected packages: pymsgbox, PyTweening, pyscreeze, pyautogui
  Running setup.py install for pymsgbox ... done
  Running setup.py install for PyTweening ... done
  Running setup.py install for pyscreeze ... done
  Running setup.py install for pyautogui ... done
Successfully installed PyTweening-1.0.3 pyautogui-0.9.36 pymsgbox-1.0.6 pyscreeze-0.1.13
```

**Comment by ripdog** [comment]
Yes, this one bit me too. Would be great if I didn't have to hunt that down.

**Comment by alphaCTzo7G** [comment]
@jtara1 .. this would awesome..theres another thread discussing this here: https://github.com/asweigart/pyautogui/issues/223

**Comment by chrahunt** [comment]
This was fixed in 13c83ce3, so this issue can be closed.
